test/tests/Addresses/RangeTypeTest.php

```diff
@@ -6,7 +6,7 @@
 use IPLib\Range\Type;
 use PHPUnit_Framework_TestCase;
 
-class TypeTest extends PHPUnit_Framework_TestCase
+class AddressRangeTypeTest extends PHPUnit_Framework_TestCase
```

Comment:
Why did you change the class name?
Furthermore, let's follow the PSR-4 rules: the class name must be the same as the file name (except for the leading .php file extension).